### PR TITLE
Scikit-learn wrapper for FastText model

### LINKED ISSUE #2138: Scikit learn wrapper for fasttext model
Is there any plan to make this wrapper? I want to integrate fasttext model in scikit learn pipeline.

Thanks.

### PR DESCRIPTION
**Pull request by mcemilg**
Fixes #2138 

Added wrapper for FastText model to use on scikit-learn pipeline. I use word2vec and doc2vec wrappers as guidance on implementation.

### CODE REVIEW

File: gensim/test/test_sklearn_api.py [hunk 1]
```diff
@@ -1213,5 +1214,91 @@ def testModelNotFitted(self):
...
+        self.model = FTTransformer(size=10, min_count=0, seed=42)
+        self.model.fit(texts)
+
+        # training a Gensim FastText model with the same params
+        gensim_ftmodel = models.FastText(texts, size=10, min_count=0, seed=42)
+
+        word = texts[0][0]
+        vec_transformer_api = self.model.transform(word)  # vector returned by FTTransformer
+        vec_gensim_model = gensim_ftmodel[word]  # vector returned by FastText
+        passed = numpy.allclose(vec_transformer_api, vec_gensim_model, atol=1e-1)
```
[the remark this answers is not in the document]
Comment (mcemilg) [reply gensim/test/test_sklearn_api.py]: I saw it on other consistency tests close to word vectors and I felt this is needed. Actually, it is passing without any tolerance parameter.

[the remark this answers is not in the document]
Comment (menshikh-iv) [reply gensim/test/test_sklearn_api.py]: In this case please remove it

**Comment by mcemilg** [comment]
Hi for people I boder. There is a wrong git ops over here because of gitkraken. I will undo last changes I did. I am sorry.

File: gensim/test/test_sklearn_api.py [hunk 2]
```diff
@@ -1213,5 +1214,91 @@ def testModelNotFitted(self):
...
+        self.assertEqual(model_params["negative"], 20)
+        # verify that the attributes values are also changed for `gensim_model` after fitting
+        self.model.fit(texts)
+        self.assertEqual(getattr(self.model.gensim_model, 'negative'), 20)
+
+    def testPersistence(self):
+        model_dump = pickle.dumps(self.model)
+        model_load = pickle.loads(model_dump)
+
+        word = texts[0][0]
```
[the remark this answers is not in the document]
Comment (menshikh-iv) [reply gensim/test/test_sklearn_api.py]: @mcemilg still here, please don't forget to fix

[the remark this answers is not in the document]
Comment (mcemilg) [reply gensim/test/test_sklearn_api.py]: Just fixed it, thanks.

File: gensim/test/test_sklearn_api.py [hunk 3]
```diff
@@ -1213,5 +1214,111 @@ def testModelNotFitted(self):
         self.assertRaises(NotFittedError, phrases_transformer.transform, phrases_sentences[0])
 
 
+class TestFastTextWrapper(unittest.TestCase):
+    def setUp(self):
+        numpy.random.seed(0)
```
Comment (menshikh-iv) [inline gensim/test/test_sklearn_api.py]: `numpy.random.seed(0)` affect all interpreter in general (not only current test), that's bad practice (I think we have similar mistakes in existing tests), can you please remove all calls of `numpy.random.seed` in your code @mcemilg ?

Comment (mcemilg) [reply gensim/test/test_sklearn_api.py]: Okay, I removed the `numpy.random.seed` calls.

**Comment by menshikh-iv** [comment]
Thanks @mcemilg, congratz with your first contribution :1st_place_medal:

**Comment by mcemilg** [comment]
Hi @menshikh-iv, thank you for your helps. Do you want me to fix [this](https://github.com/RaRe-Technologies/gensim/pull/2178#discussion_r218274113) issue? If you want I can work on this issue.

**Comment by menshikh-iv** [comment]
@mcemilg If you have time for it  - of course, I will be very grateful :)

**Comment by mcemilg** [comment]
Okay, I will look it as soon as possible. :+1:

**Comment by menshikh-iv** [comment]
@mcemilg note: global seeding should never happen (i.e have a look through all library, not `test` only).
